src/routes/Send.tsx

```diff
@@ -192,7 +193,18 @@ function DestinationShower(props: {
             <Match when={props.nodePubkey && props.source === "lightning"}>
                 <StringShower text={props.nodePubkey || ""} />
             </Match>
-            <Match when={props.lnurl && props.source === "lightning"}>
+            <Match
+                when={props.lightning_address && props.source === "lightning"}
+            >
+                <StringShower text={props.lightning_address || ""} />
```

Comment:
string shower is meant for longer strings, it chokes on longish lightning addresses

![Screenshot 2023-11-17 at 1 53 51 PM](https://github.com/MutinyWallet/mutiny-web/assets/543668/3cf9c21c-9d86-4b71-b8be-d5da96aef596)

I can fix if you'd like

Reply:
yeah can do you do that

Reply:
didn't end up with a truly beautiful solution but it works... seems like the long term perfect solution to a middle ellipses is some container-measuring shenanigans

Reply:
cool update looks good to me